PROJ.lua

```diff
@@ -0,0 +1,51 @@
+project "PROJ"
+
+dofile(_BUILD_DIR .. "/static_library.lua")
+
+configuration { "*" }
+
+uuid "E63D9AD5-9F32-4772-B710-38EFF074067C"
```

Comment:
Jut double checking that this is unique and wasn't copy pasted?

Reply:
I regenerated it just to be sure.